docker-compose.yml

```diff
@@ -9,3 +9,27 @@ services:
       - "8080:8080"
     env_file:
       - ./api/.env
+
+  db:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    ports:
+      - "27017:27017"
+
+  db-seed:
+    build: ./db/seed
```

Comment:
Sorry, my knowledge of docker is quite sparse, so this might be a silly question: should the 'build' property be using the path from the project or from the image being built? I ask because this is the path to the seed file in the project, but I don't think you're copying the 'db' and 'seed' folders into the image.

Reply:
It should be using the project path, yes. The `build` property points to the Dockerfile, which then builds the image from there - the folders never make it to the image, as this is all pre-image construction